src/index.tsx

```diff
@@ -679,25 +660,43 @@ export class Modalize<FlatListItem = any, SectionListItem = any> extends React.C
   };
 
   private renderModalize = (): React.ReactNode => {
-    const { modalStyle, adjustToContentHeight, keyboardAvoidingBehavior, alwaysOpen } = this.props;
+    const {
+      keyboardAvoidingOffset,
+      modalStyle,
+      keyboardAvoidingBehavior,
+      alwaysOpen,
+      panGestureEnabled,
+    } = this.props;
     const { isVisible, lastSnap, showContent } = this.state;
-    const enabled = isIos && adjustToContentHeight;
+    const enabled = isIos;
     const pointerEvents = alwaysOpen ? 'box-none' : 'auto';
 
+    const keyboardAvoidingViewProps: KeyboardAvoidingViewProps = {
+      keyboardVerticalOffset: keyboardAvoidingOffset,
+      behavior: keyboardAvoidingBehavior || 'padding',
+      enabled,
+      style: [s.modalize__content, this.modalizeContent, modalStyle],
+    };
+
+    if (!isIos) {
```

Comment:
Why are you not running the function on ios?

Reply:
Because if pass onLayout prop to KAW on iOS, it overrides onLayout logic in KAW and it stops working(RN bug) https://github.com/facebook/react-native/blob/8553e1acc4195479190971cc7a3ffaa0ed37a5e0/Libraries/Components/Keyboard/KeyboardAvoidingView.js#L119